test/distributed/_tensor/experimental/test_tp_transform.py

```diff
@@ -42,6 +49,13 @@ def test_tensor_parallel_transformation_e2e(self):
         torch.manual_seed(0)
         model = MLPListModule(2).to(device=self.device_type)
         inputs = (torch.randn((10, 12)).to(device=self.device_type),)
+        parallel_strategies: Dict[str, ParallelStyle] = {
```

Comment:
can you also add a test where the mlps are created with `bias=False`? To verify that with no bias the pass should also work

Reply:
Absolutely! Good catch